expenses/views.py

```diff
@@ -125,6 +125,9 @@ def edit_expense(request, pk):
     expense.description = request.POST['description']
     expense.expense_date = request.POST['expense_date']
     expense.is_digital = 'is-digital' in request.POST
+    if (expense.is_flagged != None):
```

Comment:
```suggestion
    if expense.is_flagged != None:
```
python doesn't need parens